mods/d2k/rules/aircraft.yaml

```diff
@@ -1,4 +1,4 @@
-carryall.scripted:
+carryall.reinforce:
```

Comment:
I meant that only the carryall used for the harvester insurance should have `TargetableAircraft` removed from it, not all of the carryalls. (Changed my mind about `carryall.infantry`, it's better if people can defend against them).  Regular Carryalls should of course be targetable.